src/Core/src/Container.php

```diff
@@ -503,7 +473,7 @@ private function createInstance(string $class, array $parameters, string $contex
 
             $instance = null;
             try {
-                /** @var InjectorInterface $injectorInstance */
+                /** @var InjectorInterface|mixed $injectorInstance */
```

Comment:
added union `/** @var InjectorInterface|mixed $injectorInstance */` to make `if (!$injectorInstance instanceof InjectorInterface) {` next valid.